includes/ColorsManager.hpp

```diff
@@ -188,7 +195,7 @@ class ColorsManager : public others::NotCopiable
     static constexpr sf::Uint8 COLOR_BLACK_BLUE = 0;
```

Comment:
colorGrey and colorDarkGrey seem unused... also check for all the colors and delete the useless ones